### PR TITLE
Fix docker compose

### PR DESCRIPTION
**Pull request by eltenedor**
There seem to be two issues that come up with the current `docker-compose` routine, i.e. when creating the container via:
```bash
sudo docker build --tag webwork-base:forWW217 -f DockerfileStage1 . && sudo docker-compose build && sudo docker-compose up
```
* Syntax: Missing backtick in `docker-entrypoint.sh`
```
app_1  | /usr/local/bin/docker-entrypoint.sh: line 180: unexpected EOF while looking for matching ``'
```
* Dependencies: `docker-compose` complains about `Email::Stuffer` missing
```
app_1  | AH00526: Syntax error on line 60 of /etc/apache2/conf-enabled/webwork.conf:
app_1  | Can't locate Email/Stuffer.pm in @INC (you may need to install the Email::Stuffer module) (@INC contains: /opt/webwork/pg/lib /opt/webwork/webwork2/lib /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.30.0 /usr/local/share/perl/5.30.0 /usr/lib/x86_64-linux-gnu/perl5/5.30 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.30 /usr/share/perl/5.30 /usr/local/lib/site_perl /etc/apache2) at /opt/webwork/webwork2/lib/WeBWorK/ContentGenerator/ProblemUtil/ProblemUtil.pm line 47.\nBEGIN failed--compilation aborted at /opt/webwork/webwork2/lib/WeBWorK/ContentGenerator/ProblemUtil/ProblemUtil.pm line 47.\nCompilation failed in require at /opt/webwork/webwork2/lib/WeBWorK/ContentGenerator/Problem.pm line 19.\nBEGIN failed--compilation aborted at /opt/webwork/webwork2/lib/WeBWorK/ContentGenerator/Problem.pm line 19.\nCompilation failed in require at /usr/share/perl/5.30/base.pm line 137.\n\t...propagated at /usr/share/perl/5.30/base.pm line 159.\nBEGIN failed--compilation aborted at /opt/webwork/webwork2/lib/WeBWorK/ContentGenerator/ShowMeAnother.pm line 18.\nCompilation failed in require at (eval 420) line 1.\nBEGIN failed--compilation aborted at /opt/webwork/webwork2/lib/WeBWorK.pm line 88.\nCompilation failed in require at /opt/webwork/webwork2/lib/Apache/WeBWorK.pm line 37.\nBEGIN failed--compilation aborted at /opt/webwork/webwork2/lib/Apache/WeBWorK.pm line 37.\n
```

### CODE REVIEW

File: DockerfileStage1 [hunk 1]
```diff
@@ -17,7 +17,7 @@ ENV DEBIAN_FRONTEND=noninteractive \
 # Phase 2 - Ubuntu 20.04 base image + required packages
 
 # Packages changes/added for ubuntu 20.04:
-#       libcgi-pm-perl (for CGI::Cookie), libdbd-mariadb-perl
+#       libcgi-pm-perl (for CGI::Cookie), libdbd-mariadb-perl, libemail-stuffer-perl
```
Comment (drgrice1) [inline DockerfileStage1:20]: Also please delete.

File: Dockerfile [hunk 2]
```diff
@@ -98,7 +98,7 @@ ENV WEBWORK_ROOT=$APP_ROOT/webwork2 \
 # Phase 3 - Ubuntu 20.04 base image + required packages
 
 # Packages changes/added for ubuntu 20.04:
-#       libcgi-pm-perl (for CGI::Cookie), libdbd-mariadb-perl
+#       libcgi-pm-perl (for CGI::Cookie), libdbd-mariadb-perl, libemail-stuffer-perl
```
Comment (drgrice1) [inline Dockerfile:101]: This is not your fault that these comments are here, but these comments do not make any sense.  Please delete them.  The comment say these are "packages changes/added" for ubuntu 20.04.  Disregarding the poor grammar what does this mean?  Changed or added from what?

My real point is that at one point the docker build was designed for Ubuntu 18.04, and later is was updated to use Ubuntu 20.04.  That is what these comments are about.  However, these comments are not comments that should have ever made it into the code.  They are pull request conversation comments as they only make sense in that context.  @taniwallach take note.

File: DockerfileStage1 (hunk 1 above)
Comment (drgrice1) [inline DockerfileStage1:20]: Also, please delete.

**Review by drgrice1 — APPROVED** [review]
This looks good.

File: Dockerfile (hunk 2 above)
Comment (eltenedor) [reply Dockerfile]: Ok, I deleted the comment and the line above it in my newest commit.

Comment (taniwallach) [reply Dockerfile]: Note taken

**Comment by drgrice1** [comment]
@taniwallach:  I am not sure that it crept in.  You and @mgage had a discussion about that.  See https://github.com/openwebwork/webwork2/pull/1435#discussion_r664815734.

**Comment by taniwallach** [comment]
> @taniwallach: I am not sure that it crept in. You and @mgage had a discussion about that. See [#1435 (comment)](https://github.com/openwebwork/webwork2/pull/1435#discussion_r664815734).

Sorry - you are correct. I did not review the history / conversation carefully, just tried to determine in which PR where it came in. I even was the one to merge that PR, as I guess @mgage convinced me that the approach was reasonable, and users could comment it out if the delay was annoying.

As someone who starts/stops Docker containers frequently, both when doing development and for maintenance on productions servers running WW inside Docker - I think keeping container startup time down is quite important.

A better long term solution might be to add conditional code so that extra packages like `nano` would be installed as part of the image build process, which would leave the `ADD_APT_PACKAGES` as a fallback approach when something is needed only short-term in the "next" running container.

Let's get these urgent fixes merged, and decide later about how to better add packages helpful for developers and not mission critical on development servers.

**Pull request merged**